### DISCUSSION TITLE
Oathkeeper Hydrator issue

### DISCUSSION THREAD

**Discussion by Mshivam2409** [ory/oathkeeper#672] [Q&A] [answered]
Hello everyone,
I am unable to use the Oathkeeper Hydrator and am not able to understand the reason I;m getting 
```
"error":"invalid character '}' looking for beginning of value"
```
Even though I have a simple code in express
```js
import express, { json } from "express"

const hydrator = express()

hydrator.use(json())

hydrator.all("/", (req, res, next) => {
    return res.status(200).json(req.body)
})

hydrator.listen(9004)
```

Even in Go with fiber framework I'm getting the same error
```go
func Hydrate(ctx *fiber.Ctx) error {
	log.Print(ctx)
	a := &models.AuthenticationSession{}
	err := ctx.BodyParser(a)
	if err != nil {
		log.Print(err)
		return err
	}
	a.Header.Add("role", "user")
	log.Print(a.Header)
	n := *a
	return ctx.Status(200).JSON(n)
}
```

**Comment by vinckr** [comment, marked as answer]
Sorry for the late reply! 
I can't say what the issue is here, maybe could you share a repo or some steps how to reproduce?

**Reply by vinckr** [reply]
Hey @Mshivam2409 ,
I will mark this as answered for now, but feel free to come back to the discussion any time.

Thanks a ton! 🐝
